tests/test_basics.py

```diff
@@ -1,17 +1,17 @@
+import logging
 import os
 import unittest
-from unittest.mock import mock_open, patch, ANY, Mock
-import logging
+from typing import Mapping, Any
+
+from unittest.mock import ANY, Mock, mock_open, patch
 
 from loggo import Loggo as LoggoType
 
-test_setup = dict(facility='LOGGO_TEST',
-                  ip=None,
-                  port=None,
-                  do_print=True,
-                  do_write=True,
-                  private_data=['mnemonic', 'priv'])
-Loggo = LoggoType(test_setup)
+test_setup = dict(do_write=True,
+                  log_if_graylog_disabled=False,
+                  private_data={'mnemonic', 'priv'})  # type: Mapping[str, Any]
```

Comment:
Could we use the same typehint style here as everywhere else? i.e. `test_setup: Mapping[str, Any] = dict(....`

Reply:
loggo does not seem to have this kind of typehint elsewhere. are you referring to some standard for other projects at bitpanda? if so, fix this in new issue/MR please, it's not my style guide

Reply:
Dont get what you mean. This typehint style is used in every single declaration, we have zero use of the comment style (besides for ignores which can only be done with the comment).

Reply:
discussed this outside of github, answer is, we don't yet have consensus on variable type declaration style for this project. we have differing opinions on stability of mypy, basically. so for now there are no rules.